src/python/pants/python/python_setup.py

```diff
@@ -151,7 +151,15 @@ def register_options(cls, register):
             default=None,
             advanced=True,
             fingerprint=True,
-            help="The maximum number of concurrent jobs to resolve wheels with.",
+            help=(
+                "The maximum number of concurrent jobs to resolve wheels with. You may want to "
+                "increase the maximum number of parallel jobs to potentially improve the latency "
```

Comment:
Wading into more words here in the 2nd sentence brings more confusion in my modelling of a naivev reader. Since the default is num cores but that sentence comes without that context, the naive read is to add a positive number value here bigger than `None`. That won't net a potential speedup until they go to a value greater than num cores and it will in fact likely even hurt performance at that point, even in v1. Adding even more words or axing this bit since it should go without saying esp. in an advanced option seem like the two paths. You know I prefer the latter. The remaining sentences though pull there weight.